### DISCUSSION TITLE
Annotations of hkil on projections

### DISCUSSION THREAD

**Discussion by lilarou** [mtex-toolbox/mtex#1938] [Ask Anything] [answered]
Hi MTEX community, 

I want plot the norml planes of a slip systems within a certain orientation and annotate each variant of the system with the corresponding 'hkil' 
`ori1 = orientation.byEuler(163.6*degree,84.29*degree,38.44*degree,CS);
sSHexPa =slipSystem.prismaticA(CS_Ti);
sSHexPa_sym = symmetrise(sSHexPa,'antipodal');
n1 = sSHexPa_sym.n;

r_n1 = ori1 * n1;

plot(r_n1, 'MarkerFaceColor', 'red', 'MarkerSize', 8, 'FontSize', 20, 'grid');`

I don't have any idea how to add the corresponding hkil from the Miller n1.

Thank you in advance

**Comment by kilir** [comment, marked as answer]
Hi,
how about 
``` plot(r_n1, 'label',char(n1,'cell'));``` ?
Cheers,
Rüdiger
